### DISCUSSION TITLE
Carpet not working on Fabric Server

### DISCUSSION THREAD

**Discussion by CarryingHim** [gnembon/fabric-carpet#797] [Q&A]
Hello  i have a Question  :smile:
i am Running a fabric 1.16.5 Server and installed carpet on it 
the Server starts normal and everything works fine yet i dont have Access to the /Carpet command 
or any other function from the carpet mod^^

**Comment by altrisi** [comment]
Make sure you have OP permissions in the server, those are required in order to run the `/carpet` command.

If you do, make sure Fabric and Carpet have loaded correctly by checking if the log has something like (near the top):
```
(FabricLoader) Loading game for Minecraft 1.16.5
...
[FabricLoader] Loading x mods: ..., carpet,..
```
If that's not present, then you probably didn't install Carpet or Fabric correctly. If it is, please send the log file so we can give you better help.

(if Fabric isn't present at all, maybe you didn't use the `fabric-server-launch` jar and used the regular vanilla launcher).
